tutorials/inverse/55_tfr_stc.py

```diff
@@ -0,0 +1,221 @@
+# -*- coding: utf-8 -*-
+"""
+.. _tut-tfr_stc:
+
+====================================
+Time-frequency source reconstruction
+====================================
+
+This tutorial shows how to use different MNE source estimation methods with
```

Comment:
yes let's stick to MNE methods and keep beamforming outside of this for now

Reply:
Sounds good, I split that the beamformer code into another PR